### PR TITLE
🐛 Destination Bigquery - ensure raw dataset created with migration

### PR DESCRIPTION
**Pull request by jbfbell**
If the raw dataset had not previously existed, we must create it for the migration

### CODE REVIEW

File: airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGenerator.java [hunk 1]
```diff
@@ -604,11 +604,16 @@ private String wrapAndQuote(final String namespace, final String tableName) {
...
         "v2_raw_table", streamId.rawTableId(QUOTE),
         "v1_raw_table", wrapAndQuote(namespace, tableName)
     )
     ).replace(
-        """      
+        """
+            CREATE SCHEMA IF NOT EXISTS ${raw_namespace}
+            OPTIONS(location="${dataset_location}");
+                 
             CREATE OR REPLACE TABLE ${v2_raw_table} (
```
Comment (evantahler) [inline airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGenerator.java:617]: Does this table create statement also need `OPTIONS(location="${dataset_location}")`, or will being in the dataset take care of it?

File: airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGenerator.java [hunk 2]
```diff
@@ -604,11 +604,16 @@ private String wrapAndQuote(final String namespace, final String tableName) {
...
+        "raw_namespace", StringUtils.wrap(streamId.rawNamespace(), QUOTE),
+        "dataset_location", datasetLocation,
         "v2_raw_table", streamId.rawTableId(QUOTE),
         "v1_raw_table", wrapAndQuote(namespace, tableName)
     )
     ).replace(
-        """      
+        """
+            CREATE SCHEMA IF NOT EXISTS ${raw_namespace}
+            OPTIONS(location="${dataset_location}");
```
Comment (evantahler) [inline airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGenerator.java:615]: ✅

**Review by evantahler — APPROVED** [review]
Makes sense!

File: airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGenerator.java (hunk 1 above)
Comment (jbfbell) [reply airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGenerator.java:617]: fairly confident that bigquery doesn't let you create tables in a different location than the dataset

**Pull request merged**